internal/kokoro/environment.sh

```diff
@@ -86,6 +86,9 @@ if [[ $ENVIRONMENT == *"kubernetes"* ]]; then
   mkdir -p ~/.local/bin
   mv ./kubectl ~/.local/bin
   export PATH=$PATH:~/.local/bin/
+  # install auth plugin
+  apt-get install google-cloud-sdk-gke-gcloud-auth-plugin
+  export USE_GKE_GCLOUD_AUTH_PLUGIN=True
```

Comment:
What does this control -- will it affect other tests? If so would you mind unsetting it at the end of the script?

Reply:
This ensures that kubectl uses the gcloud auth plugin, now that the previous method is deprecated

I could disable it at the end of the tests, but keep in mind that this `environment.sh` file is only used for the logging library-specific environment test entrypoint, so this shouldn't impact other tests